csharp/private/macros/setup_basic_nuget_package.bzl

```diff
@@ -73,9 +82,9 @@ def setup_basic_nuget_package():
             netstandard1_6 = tfms.get("netstandard1.6"),
             netstandard2_0 = tfms.get("netstandard2.0"),
             netstandard2_1 = tfms.get("netstandard2.1"),
-            net20 = tfms.get("net20"),
-            net40 = tfms.get("net40"),
-            net45 = tfms.get("net45"),
+            net20 = tfms.get("net20") or tfms.get("net11"),
```

Comment:
Due to the `NugetOnlyFrameworks` translation, these `or`s will never get hit.

Reply:
I still think these `or`s will never be hit. `imports` isn't touched until `net11` has been mapped to `net20`

Reply:
Alternative phrasing: These `or`s are all unnecessary.